torch/_higher_order_ops/cond.py

```diff
@@ -234,8 +234,12 @@ def trace_cond(proxy_mode, func_overload, pred, true_fn, false_fn, operands):
         isinstance(o, torch.Tensor) for o in operands
     ), "Cond operands must be a list of tensors"
 
```

Comment:
suppressing here for the make_fx call, and the function below for the collect_metadata_analysis pass. What would the more general higher-order-op place for this be?